src/cli/cli.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from 'commander';
+
+let program : Command;
+
+export const initCLI = () => {
+    program = new Command();
+    program
+    .option('-f, --filepath <string>', "A path to load a kamon game from a file")
+    .parse(process.argv);
+}
+
+export const getFilePath = ():string => {
+    const filePathParameter = program.opts()['filepath'];
+
+    return filePathParameter != null ? filePathParameter : `${__dirname}/../../save.json`
```

Comment:
Shorter syntax:

```suggestion
    return filePathParameter ?? `${__dirname}/../../save.json`
```

Also, I'm pretty sure you could have added the default value here

```diff
- .option('-f, --filepath <string>', "A path to load a kamon game from a file")
+ .option('-f, --filepath <string>', "A path to load a kamon game from a file", "default value")
```
But this is nitpicking :wink: